ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vlan_interfaces.schema.yml

```diff
@@ -0,0 +1,472 @@
+        ip_address_virtual_secondaries:
+          type: list
+          items:
+            type: str
+            description: IPv4_address/Mask
+        ip_igmp:
+          type: bool
+        ip_helpers:
+          type: list
+          description: List of DHCP servers
+          convert_types:
+          - dict
+          primary_key: ip_helper
+          items:
+            type: dict
+            keys:
+              ip_helper:
+                type: str
+                description: IP address or hostname of DHCP server
+              source_interface:
+                type: str
+                description: Interface used as source for forwarded DHCP packets
+              vrf:
+                type: str
+                description: VRF where DHCP server can be reached
+        ipv6_enable:
+          type: bool
+        ipv6_address:
+          type: str
+          description: IPv6_address/Mask
+        ipv6_address_virtual:
+          type: str
+          description: |
+            IPv6_address/Mask
+            "ipv6_address_virtual" key will be deprecated in AVD v4.0 in favor of the new "ipv6_address_virtuals"
+            If both "ipv6_address_virtual" and "ipv6_address_virtuals" are set, all addresses will be configured
+        ipv6_address_virtuals:
+          type: list
+          description: The new "ipv6_address_virtuals" key support multiple virtual ipv6 addresses.
+          items:
+            type: str
+            description: IPv6_address/Mask
+        ipv6_address_link_local:
+          type: str
+          description: IPv6_address/Mask
+        ipv6_virtual_router_address:
+          type: str
+          description: |
+            "ipv6_virtual_router_address" key will be deprecated in AVD v4.0
+            This should not be mixed with the new "ipv6_virtual_router_addresses" key below to avoid conflicts.
+        ipv6_virtual_router_addresses:
+          type: list
+          description: Improved "VARPv6" data model to support multiple VARPv6 addresses.
+          items:
+            type: str
+            description: IPv6 address or IPv6_address/Mask
+        ipv6_nd_ra_disabled:
+          type: bool
+        ipv6_nd_managed_config_flag:
+          type: bool
+        ipv6_nd_prefixes:
+          type: list
+          convert_types:
+          - dict
+          primary_key: ipv6_prefix
+          items:
+            type: dict
+            keys:
+              ipv6_prefix:
+                type: str
+                description: IPv6_address/Mask
+              valid_lifetime:
+                type: str
+                description: In seconds <0-4294967295> or infinite
+                convert_types:
+                - int
+              preferred_lifetime:
+                type: str
+                description: In seconds <0-4294967295> or infinite
+                convert_types:
+                - int
+              no_autoconfig_flag:
+                type: bool
+        ip_router_virtual_address:
```

Comment:
This is not documented in our current README. where does it come from?

Reply:
good catch, removed in [abc1968](https://github.com/aristanetworks/ansible-avd/pull/2106/commits/abc196847f3c0be991a33b6f809b74f381049682)